community.html

```diff
@@ -58,7 +58,7 @@ <h4 class="card-title">GitHub</h4>
         <h4 class="card-title">JIRA</h4>
         <p class="card-text">
           We use JIRA for issue tracking, organizing development sprints, and
-          planning our road map. View the list of <a href="https://issues.jboss.org/projects/FH/issues/FH-2744?filter=allopenissues">open issues</a>.
+          planning our road map. View the list of <a href="https://issues.jboss.org/projects/FH/issues/FH-3708?filter=allissues">open issues</a>.
```

Comment:
Thanks for feedback. Changed to `https://issues.jboss.org/projects/FH/issues/?filter=allopenissues`.